### PR TITLE
Add /talent all command to set all characters to full talent level

### PR DESCRIPTION
**Pull request by cfuncode**

### CODE REVIEW

**Comment by cfuncode** [comment]
https://github.com/Grasscutters/Grasscutter/pull/692

**Comment by cfuncode** [comment]
Relics can be added using the give all command https://github.com/Grasscutters/Grasscutter/pull/692

**Comment by ghost** [comment]
My personal opinion is that relics (or artifacts as they are called) should not be given in /giveall. Mainly because their stats are randomly generated and doing /giveall for them can cresult in a bunch of relics with useless stats.

**Comment by cfuncode** [comment]
> My personal opinion is that relics (or artifacts as they are called) should not be given in /giveall. Mainly because their stats are randomly generated and doing /giveall for them can cresult in a bunch of relics with useless stats.

It shouldn't matter. Unworn and unlocked relics can be removed using the /clear art command.

**Comment by lhhxxxxx** [comment]
> > My personal opinion is that relics (or artifacts as they are called) should not be given in /giveall. Mainly because their stats are randomly generated and doing /giveall for them can cresult in a bunch of relics with useless stats.
> 
> It shouldn't matter. Unworn and unlocked relics can be removed using the /clear art command.

Firstly, level 21 artifacts will not be cleared currently.
Secondly, what is the significance to give a backpack of random attributes artifacts?

**Comment by cfuncode** [comment]
> > > My personal opinion is that relics (or artifacts as they are called) should not be given in /giveall. Mainly because their stats are randomly generated and doing /giveall for them can cresult in a bunch of relics with useless stats.
> > 
> > 
> > It shouldn't matter. Unworn and unlocked relics can be removed using the /clear art command.
> 
> Firstly, level 21 artifacts will not be cleared currently. Secondly, what is the significance to give a backpack of random attributes artifacts?

I know, there are changes in this submission.
Because I don't want to add relics one by one, the experience is so bad that it's better to be random. 
Anyway, I've become numb to brushing relics every day. I don't like the emptiness of the holy relics backpack, the piles of holy relics give me a sense of the king's treasure house.

**Comment by lhhxxxxx** [comment]
> > > > My personal opinion is that relics (or artifacts as they are called) should not be given in /giveall. Mainly because their stats are randomly generated and doing /giveall for them can cresult in a bunch of relics with useless stats.
> > > 
> > > 
> > > It shouldn't matter. Unworn and unlocked relics can be removed using the /clear art command.
> > 
> > Firstly, level 21 artifacts will not be cleared currently. Secondly, what is the significance to give a backpack of random attributes artifacts?
> 
> I know, there are changes in this submission.
> Because I don't want to add relics one by one, the experience is so bad that it's better to be random. 
> Anyway, I've become numb to brushing relics every day. I don't like the emptiness of the holy relics backpack, the piles of holy relics give me a sense of the king's treasure house.

This is your **PERSONAL** idea, and also your commits contain many other undescribed changes. You should create another pull request to deal with these undescribed changes. Your pull request may not be merged until you remove the irrelevant commits.

File: src/main/java/emu/grasscutter/command/commands/TalentCommand.java [hunk 1]
```diff
@@ -111,4 +117,78 @@ public void execute(Player sender, Player targetPlayer, List<String> args) {
             }
         }
     }
+
+    private void setAllAvatarTalentLevelMax(Player sender, Player player) {
```
Comment (Birdulon) [inline src/main/java/emu/grasscutter/command/commands/TalentCommand.java]: This seems very excessive for something that aims to be very similar to the n/e/q cases.

**Pull request closed without merging**